pre_commit_hooks/loaderon_hooks/Documentation/automatic_testing.txt

```diff
@@ -0,0 +1,6 @@
+In order to run automatic tests and know test coverage, following commands are suggested (notice that will run
+ALL pre_commit_hooks tests, not only Loaderon's):
+
+coverage erase
+coverage run --source pre_commit_hooks -m pytest
+coverage report
```

Comment:
fwiw, I think this is just assumed -- probably don't need to repeat it here

also if you use markdown (`.md`) github will higlight it for you